components/library/libgpg-error/patches/02-include-amd64-search-path.patch

```diff
@@ -0,0 +1,14 @@
+This includes /usr/lib/amd64 to `Libs:` in the amd64 .pc file and thus breaks
```

Comment:
heals?

Reply:
Right... Fixed in https://github.com/OpenIndiana/oi-userland/pull/5278/commits/26bf75f863930f824ae83d51950c666dd9e88b70#diff-9dcdfaaa584909b13eb99142fa1eccf6.